tests/test_views.py

```diff
@@ -0,0 +1,36 @@
+from .test_base import BaseTestConfig
+from flask_login import current_user
+from models.models import User, db
+from flask import url_for
+import unittest
+
+class TestUserViews(BaseTestConfig):
+    
+    def test_index_page_loads(self):
+        response = self.client.get('/index')
+        # self.assertIn(response.data)
```

Comment:
1. commenting this out invalidates the test..i.e this is no longer a test and does not need to exist.
2. Do not push commented code.